scripts/polyfills.js

```diff
@@ -0,0 +1,11 @@
+/* eslint no-extend-native: 0 */
```

Comment:
We need to add notes here, the `polyfill` file is used for document sites rather than lib. 
(we can use documents to guide users how to use `polyfill`)

Reply:
Yes, I agree, this should be stated in the docs (with examples), so I added the polyfill for the docs just to test on IE.